common/config/rush/pnpm-config.json

```
@@ -91,6 +91,7 @@
   "globalOverrides": {
     // "example1": "^1.0.0",
     // "example2": "npm:@company/example2@^1.0.0"
+    "got": "12.5.3"
```

The current change breaks functionality. It appears that `got` is derriving only from `package-json` in the repo.

```suggestion
      "globalOverrides": {
    // "example1": "^1.0.0",
    // "example2": "npm:@company/example2@^1.0.0"
    // TODO: Remove once https://github.com/dylang/npm-check/issues/499 
    // has been closed and a new version of `npm-check` is published.
    "package-json": "^7"
  }
```

After verifying the lockfile there should be no more reference to `got` less than version 11.